docs/Maintain/Farming/Pulsar/index.mdx

```diff
@@ -0,0 +1,262 @@
+:::
+
+:::note Feedback!
+Your insights on our releases elevate our progress. Fill out this [form](https://c6gxctjlh68.typeform.com/to/Ks4Y16Fo), your voice will be heard and addressed.
+:::
+
+## Pre-Requisites
+
+---
+
+### System Requirements
+
+:::danger Farming can be Network Intensive.
+
+Make sure you have a stable network connection. During the plotting phase of farming, it can be network intensive.
+
+This may impact your network usage so please check your network connection if you have a hard data limit.
+:::
+
+Pulsar requires commodity hardware specs to operate. At a **minimum**, it is recommended to have:
+
+| Hardware | Specs            |
+| -------- | ---------------- |
+| CPU      | 4 Core+          |
+| RAM      | 8GB+             |
+| SWAP     | 4GB              |
+| Storage  | 100GB SSD        |
+
+:::note SWAP Warning
+A swap is only necessary if you have a low amount of RAM. 
+:::
+
+### Getting A Crypto Wallet
+
+Before running anything you need to have a wallet where you'll receive testnet coins.
+Follow our how-to guides in the [Wallets section](/docs/category/wallets/) of the documentation for steps on how to get your wallet setup.
+
+### Required ports
+
+Currently, a few ports need to be exposed for node to work properly.
+
+If you have a server with no firewall, there is nothing to be done, but otherwise make sure to open the following TCP and UDP ports for incoming connections.
+
+- `30333`
+- `30433`
+- `30533`
+
+On the desktop side if you have a router in front of your computer, you'll need to forward TCP and UDP ports to the machine on which your node is running (how this is done varies from router to router, but there is always a feature like this, refer to [How to Forward Ports](port-forwarding.md) for a more in-depth tutorial).
+If you're connected directly without any router, then again nothing needs to be done in such case.
+
+
+# Installation
+---
+:::caution ALPHA SOFTWARE
+Pulsar is in **alpha**.
+Please feel free to file bug reports on our GitHub issues.
+:::
+<details>
+  <summary>
+  Prefer A video? Expand for our installation video
+  </summary>
+  <div>
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/h1-CX7lGl2w" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+  </div>
+</details>
+
+---
+
+:::caution
+
+Some older processors/VMs are no longer supported by official releases, but can still be compiled manually if desired
+
+:::
+
+Precompiled versions of the Pulsar is [hosted on GitHub](https://github.com/subspace/pulsar/releases). This is the recommended way to install the application. Please find the appropriate binary for your operating system.
+
+<Tabs groupId="OS">
+
+<TabItem value="windows" label="🖼️ Windows" default>
+<WindowsGuide />
+</TabItem>
+
+<TabItem value="macos" label="🍎macOS" default>
```

Comment:
In the preview, the MacOS tab doesn't work, and it shows the Linux one instead. Might be because there should be only one "default"

Reply:
Okay, the issue is with the `_macos.mdx` and it has some Linux-related data. Fixing `default` is still a good idea.